sunpy/data/_sample.py

```diff
@@ -166,3 +173,31 @@ def download_sample_data(overwrite=False):
             _handle_final_errors(results)
 
     return results + already_downloaded
+
+
+def _get_sample_files(filename_list, no_download=False, force_download=False):
+    sampledata_dir = _get_sampledata_dir()
+
+    fullpath = [sampledata_dir/fn for fn in filename_list]
```

Comment:
```suggestion
    fullpaths = [sampledata_dir/fn for fn in filename_list]
```
Since it's multiple paths?

Reply:
Changed